test/unit/fields_definition_test.rb

```diff
@@ -1,32 +1,40 @@
+# frozen_string_literal: true
+
 require 'test_helper'
 
+def new_field_definition(**opts)
+  FieldsDefinition.new(account: @provider, target: 'User', name: 'new_field', label: 'lal', **opts)
+end
```

Comment:
why is this outside classes? Can it be in one of the classes or in a helper class or something?

Reply:
I don't think this should go in a helper, it's too silly of a method. I would put it all inside a `FieldsDefinitionTest` class or module, but then I'd need to rename the current [`FieldsDefinitionTest`](https://github.com/3scale/porta/pull/2799/files#diff-e8951b1c6f578bd6a73ca3802c1353151dbc036e0e630e6e69ab87875f8cc015R9) to something like `OtherFieldsDefinitionTest` 🤮

Reply:
What you have done seems reasonable to me. While you can define a new sub-class with the method and have other sub-classes inherit from it, naming would again be tricky. btw I think you can make the method `private`, not that it matters in this case.